File: app/src/main/java/com/example/sampledax/ui/dax/data/response/UnderlyingEntity.java

package com.example.sampledax.ui.dax.data.response;


import com.google.gson.annotations.SerializedName;


@SuppressWarnings("unused")
public class UnderlyingEntity {

    @SerializedName("name")
    private String mName;
    @SerializedName("type")
    private String mType;
    @SerializedName("urlName")
    private String mUrlName;
    @SerializedName("value")
    private String mValue;

    public String getName() {
        return mName;
    }

    public void setName(String name) {
        mName = name;
    }

    public String getType() {
        return mType;
    }

    public void setType(String type) {
        mType = type;
    }

    public String getUrlName() {
        return mUrlName;
    }

    public void setUrlName(String urlName) {
        mUrlName = urlName;
    }

    public String getValue() {
        return mValue;
    }

    public void setValue(String value) {
        mValue = value;
    }

}

File: app/src/main/java/com/example/sampledax/ui/dax/data/response/UnderlyingEntity.kt
package com.example.sampledax.ui.dax.data.response

import com.google.gson.annotations.SerializedName


@Suppress("unused")
class UnderlyingEntity {
    @SerializedName("name")
    var name: String? = null

    @SerializedName("type")
    var type: String? = null

    @SerializedName("urlName")
    var urlName: String? = null

    @SerializedName("value")
    var value: String? = null
}
